### PR TITLE
Reorganize tabs and apps

### PR DESCRIPTION
**Pull request by domoritz**

### CODE REVIEW

File: site/usage/applications.md [hunk 1]
```diff
@@ -7,41 +7,53 @@ permalink: /applications.html
...
+-   <span class="octicon octicon-star"></span> [Elm-Vega](http://package.elm-lang.org/packages/gicentre/elm-vega/latest) generates Vega-Lite specifications in the pure functional language [Elm](http://elm-lang.org).
+-   [ipyvega](https://github.com/vega/ipyvega) supports Vega and Vega-Lite charts in Jupyter Notebooks.
 -   [Vega-Lite for Julia](https://github.com/fredo-dedup/VegaLite.jl)
 -   [Vega-Lite "bindings" for R](https://github.com/hrbrmstr/vegalite), create Vega-Lite visualizations in R.
 -   [vegaliteR](https://github.com/timelyportfolio/vegaliteR), vega-lite htmlwidget for R.
 -   [Vegas](https://github.com/aishfenton/Vegas) brings visualizations to Scala and Spark using Vega-Lite.
 -   [Vizard](https://github.com/yieldbot/vizard) provides a tiny REPL client to visualize Clojure data in the browser.
 
-## Tools
+## Tools where you can use Vega-Lite
```
Comment (kanitw) [inline site/usage/applications.md]: Platforms that?

File: site/usage/applications.md [hunk 2]
```diff
@@ -7,41 +7,53 @@ permalink: /applications.html
...
--   [PdVega](https://jakevdp.github.io/pdvega/), lets you create interactive Vega-Lite plots for Pandas.
 -   [nteract](https://github.com/nteract/nteract), interactive notebook application with Vega and Vega-Lite renderer.
--   [Observable](https://beta.observablehq.com/), an interactive JavaScript notebook.
+-   <span class="octicon octicon-star"></span> [Observable](https://beta.observablehq.com/), an interactive JavaScript notebook. [Example](https://beta.observablehq.com/@mbostock/exploring-data-with-vega-lite).
+-   [data.world](https://data.world), upload `.vg.json` and `.vl.json` files along side your raw data, or [embed Vega](https://docs.data.world/tutorials/markdown/#vega-and-vega-lite) directly into comments and summary markdown.
+-   <span class="octicon octicon-star"></span> [vega-desktop](https://github.com/kristw/vega-desktop), a desktop app that let you open `.vg.json` and `.vl.json` to see visualizations just like you open image files with an image viewer. Can also be used for [creating visualizations with Vega/Vega-Lite locally](https://medium.com/@kristw/create-visualizations-with-vega-on-your-machine-using-your-preferred-editor-529e1be875c0).
```
Comment (kanitw) [inline site/usage/applications.md]: Vega desktop is a bit different from the rest up here?

File: site/usage/applications.md [hunk 3]
```diff
@@ -7,41 +7,53 @@ permalink: /applications.html
...
+
+## Authoring Tools
+
+-   <span class="octicon octicon-star"></span> [Vega-Editor](https://vega.github.io/editor/), the editor for Vega and Vega-Lite.
+
+## Tools that use Vega-Lite
+
+-   <span class="octicon octicon-star"></span> [Voyager (2)](https://github.com/vega/voyager), visualization tool for exploratory data analysis that blends a Tableau-style specification interface (formerly [Polestar](https://github.com/vega/polestar)) with chart recommendations (formerly the Voyager visualization browser).
```
Comment (kanitw) [inline site/usage/applications.md]: Voyager is like a GUI for visual authoring Vega-Lite, so it's a bit different from the rest?  

I'd like to think that Vega-Desktop, Vega-Editor, Voyager are all tools for using Vega-Lite?

File: site/usage/applications.md (hunk 1 above)
Comment (domoritz) [reply site/usage/applications.md]: Not really. Better: Application

File: site/usage/applications.md (hunk 2 above)
Comment (domoritz) [reply site/usage/applications.md]: So, where should it go?

**Comment by domoritz** [comment]
I merged two categories as it seems like an odd distinction.

**Pull request merged**